style/styles.css

```diff
@@ -453,6 +451,128 @@ header {
   margin-top: 6px;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 50px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
```

Comment:
- [ ] IN addition to what you have done, **please make your popup responsive to small device screens**. This is because I realized that it is not responsive **while it is the project requirements.** Moreover, you have been provided with the design of how your popup should look on mobile screens. Please make your popup responsive 😃 🏃
**Your Work**
![image](https://user-images.githubusercontent.com/90222110/180577754-7ad173d0-d47d-4034-99bc-d2f66473d03a.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/180577808-5e881b50-d133-4057-ad31-8b6a06e4227a.png)